### ISSUE TITLE
Passing bootstrap parameter to CatBoostClassifier fails:

### ISSUE THREAD

**Issue by Justin-Tan** [catboost/catboost#263]
When passing the `bootstrap_type` parameter to the catboost classifier, e.g.
```
model = CatBoostClassifier(iterations=2, depth=2, learning_rate=1, loss_function='Logloss', logging_level='Verbose', bootstrap_type='Poisson')
```

It fails and returns the error
`CatboostError: Invalid param `bootstrap_type`.`

Has the name of the parameter changed? I am using v0.6.2 on Python 3.6.4.

**Comment by Noxoomo** [comment]
Hi!

We forgot to add this option to python-package. We'll fix it soon.

JFYI: Poisson bootstrap-type  supported on GPU only, CPU currently supports bernoulli or bayesian only

**Comment by sab** [comment]
Fixed in 0.6.3, https://github.com/catboost/catboost/releases
